MetabolicAtlas/MetabolicAtlas
# feat: enable all data types for custom overlays

**Related issue(s) and PR(s)**  
This PR closes #1125.



**Type of change**  

 
**List of changes made**  

- Allow other data types (currently just `metabolomics`) to be selected for custom overlay data.

**Screenshot of the fix**  

<img width="1122" alt="image" src="https://user-images.githubusercontent.com/423498/194089916-29fe216f-7120-4fa5-8e68-ab652d961ceb.png">

**Testing**  

- Relative urls that can be reused both for production and local testing
- Instructions on how to test

Mock data example for `metabolomics`:

```csv
id	tissue 1	tissue 2
MAM02680m	0.6	0.3
MAM01628c	0.55	0.9
```

Mock data example for `transcriptomics`:

```csv
id	tissue 1	tissue 2
MAR01591	0.6	0.3
MAR00447	0.55	0.9
```

1. Create a couple of mock custom data files using the snippet above.
3. Use the mock custom data files to test the following scenarios. Make sure that 
     - the nodes as colored as intended, especially for `metabolomics` since it is new
     - you can remove uploaded custom files and overlay cards
     - you can copy the url, open it in another tab and that nodes are colored as expected

     i. Single overlay card, single mock file
     ii. Single overlay card, multiple mock files of different types
     iii. Single overlay card, multiple mock files of same types
     iiii. Multiple overlay cards, single mock file
     iiiii. Multiple overlay cards, multiple mock files of different types
     iiiiii. Multiple overlay cards, multiple mock files of same type
     
    Try this out on both MapViewer (2D and 3D) as well as IP. Suggested links:
    -  `/explore/Human-GEM/map-viewer/coa_synthesis?dim=2d`
    - `/explore/Human-GEM/interaction-partners/MAM02680m`


**Further comments**  
Please come up with other testing scenarios as well!


**Definition of Done checklist**
